tests/integration/federation/test.sh

```diff
@@ -127,7 +127,16 @@ function show_workload_status() {
   fi
 
   echo "cofidectl workload status successful"
-  exit 0
```

Comment:
This was previously exiting early on success

Reply:
This is a very good catch. We have the same issue in the single-trust-zone test.

Reply:
Just to be clear, the `exit 0` needs to also be removed from tests/integration/single-trust-zone/test.sh